garrysmod/lua/includes/extensions/util.lua

```diff
@@ -365,3 +365,33 @@ function util.RemovePData( steamid, name )
 	sql.Query( "DELETE FROM playerpdata WHERE infoid = " .. SQLStr( name ) )
 
 end
+
+--[[---------------------------------------------------------
+	Name: BinaryModuleInstalled( name )
+	Desc: Returns whether a binary module with the given name is present on disk
+-----------------------------------------------------------]]
+do
+	local suffix = ({"osx64","osx","linux64","linux","win64","win32"})[
+		( system.IsWindows() && 4 || 0 )
+		+ ( system.IsLinux() && 2 || 0 )
+		+ ( jit.arch == "x86" && 1 || 0 )
+		+ 1
+	]
+	local fmt = "lua/bin/gm" .. (CLIENT && "cl" || "sv") .. "_%s_%s.dll"
+	function util.BinaryModuleInstalled( name )

```

Comment:
Shouldn't it be `util.IsBinaryModuleInstalled`?

Reply:
Yes... it should 🤔